packages/block-library/src/query-total/block.json

```diff
@@ -40,6 +40,13 @@
 			"__experimentalDefaultControls": {
 				"fontSize": true
 			}
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true
 		}
-	}
+	},
+	"style": "wp-block-query-total"
```

Comment:
Not sure this is required as [it's automatically set](https://github.com/WordPress/wordpress-develop/blob/27bb7ac2349eb81eb4e0ff5d275ff4ff3b98f3c4/src/wp-includes/blocks/index.php#L125-L127).

But there's no harm in leaving it.

Reply:
I think this is fine for now. In the future, we may be able to remove the `style` and `editrStyle` fields from all core blocks.